FogCreek/Glitch-Community
# Ambiguous map: CoverContainer

## Links
* https://ambiguous-map.glitch.me/
* https://glitch.manuscript.com/f/cases/3328228/

## GIF/Screenshots:
![Screen Shot 2019-04-10 at 4 48 36 PM](https://user-images.githubusercontent.com/6620164/55912430-a4df9a80-5bb0-11e9-8392-310a5058784d.png)
![Screen Shot 2019-04-10 at 4 48 48 PM](https://user-images.githubusercontent.com/6620164/55912431-a4df9a80-5bb0-11e9-88ea-e7699f3629ad.png)
![Screen Shot 2019-04-10 at 4 49 16 PM](https://user-images.githubusercontent.com/6620164/55912432-a4df9a80-5bb0-11e9-8724-f378f5a9b87f.png)

## Changes:
* ....hopefully none! Replaces the CoverContainer with a Component, based on SearchResultCoverBar since it does a similar thing with the coverStyle. 

## How To Test:
check out https://ambiguous-map.glitch.me/ and look at:
* user page
* team page
* project page (see it used for More by Team and More by User)
* home page when you are signed in
* try uploading a cover image and clearing a cover image
* check out https://ambiguous-map.glitch.me/storybook

## Feedback I'm looking for:
* any ui regressions I might have missed?

## Things left to do before deploying:
- [x] cross browser test
